dashboard/app/models/user.rb

```diff
@@ -245,6 +246,21 @@ class User < ApplicationRecord
 
   before_destroy :soft_delete_channels
 
+  before_validation on: :create, if: -> {gender_input_type.present?} do
+    # Limit the gender to 100 characters, this should be sufficient for all languages.
```

Comment:
I chose 100 because the upper limit is 4000 in our Redshift database column, and 100 seemed like a generous amount to me. If you think there is a better number I could go with, please suggest.